commands/accountabilityUnpin.js

```diff
@@ -0,0 +1,16 @@
+const accountabilityActions = require('../eventActions/accountabilityActions');
+const config = require('../config.json');
+
+module.exports.execute = async (client, message) => {
+	if(message.channel.id === config.channels.accountability) {
+		// Call to method elsewhere
+		accountabilityActions.userUnpinsMessage(message, message.author);
+	}
+};
+
+module.exports.config = {
+	name: 'accountabilityUnpin', // Chose this name for organizational sake
+	aliases: ['accountabilityUnpin', 'unpin'],
+	description: 'I will unpin a message for you in accountability',
```

Comment:
Is it a random message? Or your latest pinned?

Reply:
It's the latest one- I'll go ahead and update that description to be more specific.

To specify, it goes through the list starting with the most recent pin and removes the first one it finds from a user. Convenient!